data/schema/server_history.proto

```
@@ -0,0 +1,23 @@
+syntax = "proto2";
+package efa.v2;
```

Buf (1.71.0)

[error] Files with package "efa.v2" must be within a directory "efa/v2" relative to root but were in directory "data/schema".

(PACKAGE_DIRECTORY_MATCH)